src/components/cart/cart-item/cart-item.tsx

```diff
@@ -67,12 +70,42 @@ export const CartItem = ({
                 <div className={styles.productInfo}>
                     <div className={styles.productNameAndPrice}>
                         <div className={styles.productName}>{productName}</div>
+
                         {item.fullPrice?.formattedConvertedAmount && (
                             <ProductPrice
                                 price={item.fullPrice?.formattedConvertedAmount}
                                 discountedPrice={item.price?.formattedConvertedAmount}
                             />
                         )}
+
+                        {item.descriptionLines && item.descriptionLines.length > 0 && (
+                            <div className={styles.options}>
+                                {item.descriptionLines
+                                    .slice(0, moreOptionsExpanded ? undefined : MAX_OPTIONS_VISIBLE)
+                                    .map((option) => (
+                                        <div key={option.name!.translated} className="paragraph3">
+                                            {option.name!.translated}:{' '}
+                                            {option.colorInfo
+                                                ? option.colorInfo.translated
+                                                : option.plainText?.translated}
+                                        </div>
+                                    ))}
+
+                                {item.descriptionLines.length > MAX_OPTIONS_VISIBLE && (
+                                    <button
+                                        className={styles.moreOptionsButton}
+                                        onClick={() => setMoreOptionsExpanded((p) => !p)}
+                                    >
+                                        {moreOptionsExpanded ? 'Less Details' : 'More Details'}
+                                        <DropdownIcon
+                                            className={classNames(styles.moreOptionsIcon, {
+                                                [styles.rotated]: moreOptionsExpanded,
```

Comment:
maybe assign `expanded` class to the `options` and use it to rotate the icon?
I mean `options.expanded .moreOptionsIcon: {transform: rotate(180deg);}`.. no obvious benefit but feels a bit more descriptive in my opinion.